packages/playwright/src/index.ts

```diff
@@ -122,6 +124,18 @@ export default class AxeBuilder {
     return this;
   }
 
+  /**
+   * Use frameMessenger with <same_origin_only>
+   *
+   * This disables use of axe.runPartial() which is called in each frame, and
+   * axe.finishRun() which is called in a blank page. This uses axe.run() instead,
+   * but with the restriction that cross-origin frames will not be tested.
+   */
+  public setLegacyMode(legacyMode = true): this {
```

Comment:
```suggestion
  public setLegacyMode(legacyMode = true): AxeBuilder {
```

Reply:
Should be `this`. If we extend this class (which we do in devtools), `AxeBuilder` is actually wrong.